wasmcloud_host/chart/Chart.yaml

```diff
@@ -17,4 +17,4 @@ type: application
 
 version: 0.6.10
 
-appVersion: "0.61.0"
+appVersion: "0.62.0"
```

Comment:
I think we're supposed to wait to bump the chart until after the host has been released

Reply:
Let me add a comment in the next one, Taylor said you can bump this but not the chart version